spring-data-rest/pom.xml

```diff
@@ -33,6 +33,15 @@
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-autoconfigure</artifactId>
+        </dependency>
+        <!--<dependency>-->
```

Comment:
Commenting this out confuses me. Can you explain?

Reply:
The project is by default working on H2 Database. So leaving this commented. We can uncomment if we want to use the SQLite database.